lib/ui/component/alert_filter_bar.dart

```diff
@@ -59,15 +58,15 @@ class _AlertFilterBarState extends State<AlertFilterBar> {
                       });
                     },
                   ),
-                  SizedBox(height: 10),
+                  const SizedBox(height: 10),
                   ElevatedButton(
                     onPressed: () {
-                      widget.onFilterChanged(_currentOrder);
```

Comment:
I think this was a typo. That's why I thought only "filter" callback is provided. Now I fixed it to call onOrderChanged instead of onFilterChanged. cc: @WenjiuWang